concepts/observability-strategy/strategy.md

```diff
@@ -0,0 +1,77 @@
+
+![b](./assets/future.drawio.svg)
+
+As mentioned before, the technology and protocols for the signal collection are different for the signal types. Logs are getting tailed from container log files, metrics are getting pulled via the prometheus format usually and traces are getting pushed via OTLP. With that, also the pre-integration so that typical signals are getting collected instantly are different per type.
+That's why the concrete concepts for the different types are different and will be discussed in more detail in the individual sections:
+* [Concept - Configurable Logging](./configurable-logging/README.md) 
+* [Concept - Configurable Monitoring](./configurable-monitoring/README.md)
+* [Concept - Configurable Tracing](./configurable-tracing/README.md)
+
+All concepts will follow general rules and will provide harmonized user APIs:
+- **By default signals of user workload will be collected**: When a workload gets instrumented as outlined in the best practice, the telemetry data will be instantly available in a pipeline without further configuration, just define a pipeline with an output and it works
+- **One input protocol per type**: Only one common input protocol (like OTLP) will be supported per signal type having best practices on how to integrate with.
+- **Custom input protocols can be integrated**: When a workload is not using the supported protocol for signal exposure, there is a way to have custom transformation to the supported protocol in place, usually related with additional operational effort (like running a dedicated otel-collector sidecar/deployment in the users namespace)
+- **Harmonized API**: Signal pipelines are defined via well-defined Kubernetes CRDs only belonging to one group and following similar semantics.
+- **One output protocol per type**: Only one common output protocol (like OTLP) will be supported per signal type.
+- **Custom output protocols can be integrated**: there is a way to have custom transformation from the supported protocol in place, usually related with additional operational effort (like running a dedicated otel-collector sidecar/deployment in the users namespace). Furthermore, typical output plugins of the used collector technology will be offered as part of a pipeline definition, however the plugins will be offered with limited support only.
+- **Blueprints are fully integrated**: Blueprints for local backend deployments will be integrated with the used pipeline mechanism so that they work instantly
```

Comment:
```suggestion
- **Blueprints are fully integrated**: Blueprints for local backend deployments are integrated with the used pipeline mechanism, so that they work instantly.
```